llvm/lib/Target/NVPTX/NVPTXInstrInfo.td

```diff
@@ -145,6 +145,7 @@ def noPTXASUnreachableBug : Predicate<"!Subtarget->hasPTXASUnreachableBug()">;
 def doF32FTZ : Predicate<"useF32FTZ()">;
 def doNoF32FTZ : Predicate<"!useF32FTZ()">;
 def doRsqrtOpt : Predicate<"doRsqrtOpt()">;
+def useApproxLog2 : Predicate<"useApproxLog2()">;
```

Comment:
Is this predicate required? Won't all Log2 nodes be removed during legalization if useApproxLog2 is not enabled?

Reply:
I think it's not necessary actually.